index.html

```diff
@@ -4,36 +4,47 @@
   <meta charset="UTF-8">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
   <title>판다마켓</title>
+  <!-- Meta Tag -->
+  <meta property="og:url" content="sprint-mission3-pandamarket.netlify.app">
+  <meta property="og:type" content="website">
+  <meta property="og:title" content="판다마켓">
+  <meta property="og:description" content="일상의 모든 물건들을 거래해보세요">
+  <meta property="og:image" content="https://cdn.discordapp.com/attachments/1339041265727307928/1347504318554832989/img-preview.png?ex=67ceb3b0&is=67cd6230&hm=297b14382dfe0df1c1b64b777985b8e379e63411f398c6633a02a5a858c84cce&">
```

Comment:
❗️ 수정요청
이러한 og 설정이 제대로 되어 있는지 [확인](https://www.opengraph.xyz/)할 수 있는 방법이 여러가지 있습니다~
확인해보시고 meta 태그를 수정해보세요~
이미지의 경우 배포되었으니 아래처럼 배포주소로 설정하시면 된답니다~

```suggestion
  <meta property="og:image" content="https://sprint-mission3-pandamarket.netlify.app/image/img-preview.png">
```